### DISCUSSION TITLE
Additional property for articles in store.ts

### DISCUSSION THREAD

**Discussion by rasurbrand** [jef/streetmerchant#2980] [Help] [answered]
Hi guys,

I want to add a new attribute/property "Availability" next to price, model, series etc. to my articles:

What I've done so far:

Added the following to store.ts: 
`export type Availability =
  | 'good'
  | 'bad'`

Added the following to util.ts for test purposes:
`export function getTestLink(): Link {
  const link: Link = {
    brand: 'test:brand',
    cartUrl: 'https://www.example.com/cartUrl',
    model: 'test:model',
    price: 100,
    series: 'test:series',
    url: 'https://www.example.com/url',
	availability: 'good'
  };
  return link;
}`

Result when testing:
`test/util.ts:14:2 - error TS2322: Type '{ brand: "test:brand"; cartUrl: string; model: "test:model"; price: number; series: "test:series"; url: string; availability: string; }' is not assignable to type 'Link'.
  Object literal may only specify known properties, and 'availability' does not exist in type 'Link'.

14  availability: 'good'`

Can someone help me?
Where do I need to add this new "type" as well?

**Comment by Mr-Tech-13** [comment, marked as answer]
Index.ts
